### PR TITLE
Fix ide loader

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix IDE Loader.

### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/20783
fixes https://issues.redhat.com/browse/CRW-2465

### How to test this PR?

Try to start a workspace with the following devfile
https://github.com/redhat-developer/codeready-workspaces/blob/crw-2.13-rhel-8/tech-preview-devfiles/idea/latest.yaml

A workspace should start.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js [hunk 1]
```diff
@@ -17,17 +17,6 @@ export class KeycloakLoader {
         const msg = "Cannot load keycloak settings. This is normal for single-user mode.";
 
         return new Promise((resolve, reject) => {
-            try {
-                if (window.parent && window.parent['_keycloak']) {
-                    window['_keycloak'] = window.parent['_keycloak'];
-                    resolve(window['_keycloak']);
-                    return;
-                }
```
Comment (ibuziuk) [inline assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js:25]: are we excluding this part?

File: assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js [hunk 2]
```diff
@@ -17,17 +17,6 @@ export class KeycloakLoader {
         const msg = "Cannot load keycloak settings. This is normal for single-user mode.";
```
Comment (ibuziuk) [inline assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js:17]: wondering, are we still supporting the single-user mode?

File: assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js (hunk 1 above)
Comment (olexii4) [reply assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js:25]: As I understand this was implemented for old UD which set that object
https://github.com/eclipse-che/che-dashboard/blob/7.11.0/src/app/index.module.ts#L205. 

The ability to get global data from the parent frame was closed for security reasons. That is why the current UD doesn't provide this object 'window['_keycloak']' at all.

File: assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js (hunk 2 above)
Comment (olexii4) [reply assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js:17]: As I remember we do not support it

**Review by akurinnoy — APPROVED** [review]
LGTM

**Comment by olexii4** [comment]
[crw-ci-test --rebuild]

**Review by AndrienkoAleksandr — APPROVED** [review]

**Review by ibuziuk — APPROVED** [review]

File: assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js (hunk 2 above)
Comment (nickboldt) [reply assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js:17]: so then why do we need this message?

**Review by nickboldt — APPROVED** [review]
seems reasonable, especially  since others have approved. (I'm not an expert here.)

**Pull request merged**